docs/templates/snakemake/dryrun.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+snakemake \
+    --rerun-incomplete \
+    --snakefile="workflow/Snakefile" \
+    --configfile="config/config.[METHOD].yaml" \
+    --cores 4 \
```

Comment:
What does this actually mean, and do we _always_ need `4`? Perhaps better to at least add a comment

Reply:
added comment